test-clients/src/main/java/com/hedera/services/yahcli/README.md

```diff
@@ -0,0 +1,95 @@
+# Yahcli v0.0.1
+A command line interface that is able to perform the following actions against a specified network.
+
+1. Account Operations
+    1. Get Balance.
+2. System File Operations
+    1. Download all/specific file.
+    2. Upload a System file.
+3. Fees Operations
+    1. List Basic Transaction and Query fees of all/specific service.
+
+## Getting Started
+### Setting up your environment
+
+Create a folder structure like this.<br>
+![folderStructure](folderStructure.png)
```

Comment:
I think people understand the `.pem` file. But it's worth explaining the `.pass` file, which I guess contains the passphrase of the `.pem` file. Also need to explain the naming convention. Is it `account{num}`.(pem or pass)? Because what if the user wants to use account50 as a payer? Also keep in mind that most of the system accounts on mainnet are multisig accounts, so if the tool doesn't support multisig account as a payer, we need to make it clear.